### PR TITLE
Revert package/package lock changes from when instability was introduced

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tests/e2e/pact/package.json [hunk 1]
```diff
@@ -33,12 +30,15 @@
     "@types/supertest": "2.0.9",
     "@typescript-eslint/eslint-plugin": "3.2.0",
     "@typescript-eslint/parser": "3.2.0",
+    "@hapi/boom": "^9.1.0",
```
Comment (dan-jones-iw) [inline tests/e2e/pact/package.json:33]: why are these moving down :sob:

Comment (dan-jones-iw) [reply tests/e2e/pact/package.json:33]: @h is before @t in alphabet

**Review by dan-jones-iw — APPROVED** [review]

File: tests/e2e/pact/package.json (hunk 1 above)
Comment (mfjarvis) [reply tests/e2e/pact/package.json:33]: Love that you just tagged two random developers in this PR :joy:

Comment (dan-jones-iw) [reply tests/e2e/pact/package.json:33]: lol oops

**Pull request merged**